community-operators/twistlock/twistlock-console-operator.v0.0.9.clusterserviceversion.yaml

```diff
@@ -0,0 +1,235 @@
+      configuration to the specified path every 24 hours\\n# Daily, weekly, and monthly
+      snapshots are retained\\n# The exported configuration can be stored on durable
+      storage or backed up remotely with other tools\\n# Sets data recovery state
+      (enabled or disabled)\\nDATA_RECOVERY_ENABLED=true\\n# Sets the directory to
+      which Twistlock data is exported\\nDATA_RECOVERY_VOLUME=/var/lib/twistlock-backup\\n\\n#####
+      Read only containers #####\\n# Sets Twistlock containers file-system to read-only\\nREAD_ONLY_FS=true\\n\\n#####
+      Storage paths #####\\n# Sets the base directory to store Twistlock local data
+      (db and log files)\\nDATA_FOLDER=/var/lib/twistlock\\n\\n##### Docker socket
+      #####\\n# Sets the location of the Docker socket file\\nDOCKER_SOCKET=${DOCKER_SOCKET:-/var/run/docker.sock}\\n#
+      Sets the type of the docker listener (TCP or NONE)\\nDEFENDER_LISTENER_TYPE=${DEFENDER_LISTENER_TYPE:-NONE}\\n\\n####
+      SCAP (XCCDF) configuration ####\\n# Sets SCAP state (enabled or disabled)\\nSCAP_ENABLED=${SCAP_ENABLED:-false}\\n\\n####
+      systemd configuration ####\\n# Installs Twistlock as systemd service\\nSYSTEMD_ENABLED=${SYSTEMD_ENABLED:-false}\\n\\n####
+      userid configuration ####\\n# Run Twistlock console processes as root (default,
+      twistlock user account)\\nRUN_CONSOLE_AS_ROOT=${RUN_CONSOLE_AS_ROOT:-false}\\n\\n####
+      selinux configuration ####\\n# If SELINUX is enabled in dockerd, enable running
+      Twistlock console and defender with a dedicated SELINUX label\\n# See https://docs.docker.com/engine/reference/run/#security-configuration\\nSELINUX_LABEL=disable\\n\\n#############################################\\n#      High
+      availability settings\\n#############################################\\n# Only
+      to be used when the Console is deployed outside of a Kubernetes cluster\\n#
+      This native HA capability uses Mongo clustering and requires 3 or more instances\\nHIGH_AVAILABILITY_ENABLED=false\\nHIGH_AVAILABILITY_STATE=PRIMARY\\nHIGH_AVAILABILITY_PORT=8086\\n\\n\\n\\n#############################################\\n#      Twistlock
+      repository configuration\\n#############################################\\n#
+      Sets the version tag of the Twistlock containers\\n# Do not modify unless instructed
+      to by Twistlock Support\\nDOCKER_TWISTLOCK_TAG=_19_03_317\\n\""}}]'
+    capabilities: Basic Install
+spec:
+  keywords: ['twistlock', 'security', 'monitoring', 'scanning', 'runtime-security']
+  maintainers:
+  - email: jeremy@twistlock.com
+    name: Jeremy Adams, Twistlock
+  provider: 
+    name: Twistlock
+  maturity: alpha
+  links:
+  - name: Source Code
+    url: https://github.com/twistlock/sample-code/tree/master/operators/twistlock-console-helm-operator
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: TwistlockConsole
```

Comment:
Please add `displayName` and `description` to the owned customresourcedefinitions.
This is required for OLM to use this CSV